src/webgl-utils/parameters.js

```diff
@@ -92,13 +94,13 @@ export const GL_PARAMETER_SETTERS = {
   [GL.CULL_FACE]: enable,
   [GL.CULL_FACE_MODE]: (gl, value) => gl.cullFace(value),
   [GL.DEPTH_TEST]: enable,
-  [GL.DEPTH_CLEAR_VALUE]: 1,
+  [GL.DEPTH_CLEAR_VALUE]: (gl, value) => gl.clearDepth(value),
   [GL.DEPTH_FUNC]: (gl, value) => gl.depthFunc(value),
-  [GL.DEPTH_RANGE]: (gl, value) => gl.depthRange(value),
-  [GL.DEPTH_WRITEMASK]: true,
+  [GL.DEPTH_RANGE]: (gl, value) => gl.depthRange(...value),
+  [GL.DEPTH_WRITEMASK]: enable,
```

Comment:
I think this should call `gl.depthMask`. [link]( https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/depthMask).